integreat_cms/cms/templates/dashboard/_todo_dashboard_row.html

```diff
@@ -0,0 +1,28 @@
+{% load i18n %}
+<div class="flex justify-between border-t-2 border-zinc-600 p-4 w-full">
+    <div class="flex items-center">
+        <div class="mr-6">
+            <i icon-name="{% spaceless %}{% block todo_dashboard_icon %}{% endblock todo_dashboard_icon %}{% endspaceless %}"></i>
+        </div>
+        <div>
+            <a href="{% spaceless %}{% block todo_dashboard_title_link %}{% endblock todo_dashboard_title_link %}{% endspaceless %}"
+               class="text-blue-500 font-bold underline">
+                {% block todo_dashboard_title %}
+                {% endblock todo_dashboard_title %}
+            </a>
+            {# djlint:off #}
+            <span>({% translate "in total" %}
+                {% block todo_dashboard_number %}
+                {% endblock todo_dashboard_number %})</span>
+            {# djlint:on #}
```

Comment:
@JoeyStk this is not yet resolved, you only implemented the suggested changes of the super template, but not in the sub templates. This means that you completely override the block content, because you didn't include the `{{ block.super }}` parts in the sub templates.